packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -680,6 +687,11 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 				await delay(delaySeconds * 1000);
 			}
 		}
+		this.mc.logger.sendErrorEvent({
```

Comment:
Any ideas of how to catch all codepaths in which summaries happen (and fail) so we don't fail to report this situation?

Reply:
These are the only codepaths calling `SummaryGenerator.summarize(...)`